### PR TITLE
BZ change

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 050d4bbb29b6d3c125d8566240d554a9c1ac3eb2:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 26ffacec13e12ee822ee1e24cd0173a7c50df668:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/services/test_provision_stack.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ def create_template(setup_provider, provider, dialog):
     yield dialog
 
 
-@pytest.mark.meta(blockers=[BZ(1340570, forced_streams=["5.6", "upstream"])])
+@pytest.mark.meta(blockers=[BZ(1346075, forced_streams=["5.5", "5.6", "upstream"])])
```
Comment (jkrocil) [inline cfme/tests/services/test_provision_stack.py]: Does this affect AWS providers as well? If not, it should probably block only for Openstack: `@pytest.mark.meta(blockers=[BZ(1346075, forced_streams=["5.5", "5.6", "upstream"]), unblock=lambda provider: provider.type != 'openstack'])`

**Comment by dajoRH** [comment]
Lint report for commit 9883c31589279ea7d2628458c2b5192cddd7a2c3:

`cfme/tests/services/test_provision_stack.py`:
- :red_circle: Line 15:1: **F401** *'BZ' imported but unused*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6cb337380096fa235ac70e4bbecfffb67faa81f7:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by sshveta** [comment]
Changed the order of ('template_name', Input("name")), in orchestration_template.py because sometimes the test fail saying "Name can't be blank " . Even though the name field has data . 
screenshot from 2016-06-14 10 53 01

**Comment by dajoRH** [comment]
Lint report for commit eff260f420510a836f4356588aafd30684941f7f:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a54454e0b4bee96abef760eee93c23e23deb496f:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by sshveta** [comment]
(cfme_env)[sshveta@localhost cfme_tests]$ py.test --long-running -v cfme/tests/services/test_provision_stack.py 
Loading templates from trackerbot                                                                                                                                                                                  
no test annotation found in None
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.5 -- py-1.4.31 -- pytest-2.7.0 -- /home/sshveta/Work/cfme_env/bin/python
rootdir: /home/sshveta/Work/cfme_env/cfme_tests, inifile: 
plugins: wait-for
collected 2 items 

Appliance's streams: [5.6, downstream]

cfme/tests/services/test_provision_stack.py::test_retire_stack[ec2east] PASSED


Test Passes locally . The locator is correct

**Comment by mfalesni** [comment]
@sshveta Can you take a look at the retire_stack test? Since it fails

**Pull request merged**